README.md

```diff
@@ -28,6 +42,8 @@ $.ajax({
 ### Question 5
 In an AJAX POST request, what does the argument of the `.done()` callback - in the example below, that would be `data` - represent? (Hint: it's not the same as in Question 4.)
 
+The user object that you passed in to the URL in the POST request
```

Comment:
Close but not quite - in a POST request, `data` is the response from the server, and by convention it's the same object you just posted.